spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -105,8 +106,10 @@
 
       scenario 'disables the phone option and displays a warning with js', :js do
         sign_in_before_2fa
-        select 'Turkey +90', from: 'International code'
-        fill_in 'Phone', with: '+90 312 213 29 65'
+        select_country('tr')
+        input = find('#user_phone_form_phone')
+        input.send_keys ('3122132965')
```

Comment:
Should these be encapsulated in the same method? Maybe something like this:
```ruby
def select_country_and_type_phone_number(country:, number:)
  find(".selected-flag").click
  find(".country[data-country-code='#{country}']:not(.preferred)").click
  input = find('#user_phone_form_phone')
  input.send_keys (number)
end
```
And you would use it like this:
```ruby
select_country_and_type_phone_number(country: 'tr', number: '3122132965')
```

Reply:
Sure, I can give this a shot! Thanks @monfresh